version.properties

```diff
@@ -1,12 +1,12 @@
-#Wed, 23 Jun 2021 19:57:48 +0200
+#Fri, 25 Jun 2021 13:27:38 +0200
 
 # WordPress version information
-wordpress.versionName=17.6-rc-4
-wordpress.versionCode=1065
+wordpress.versionName=17.6
+wordpress.versionCode=1067
 
 wordpress.zalpha.versionName=alpha-301
 wordpress.zalpha.versionCode=1066
 
 # Jetpack version information
-jetpack.versionName=17.6-rc-3
-jetpack.versionCode=1063
+jetpack.versionName=17.6
+jetpack.versionCode=1064
```

Comment:
@jkmassel I know you advocated for jetpack and wordpress to use the same versionCode, but given that WP uses alpha builds, and thus increment the vanilla's versionCode by increment of 2 in alternance with the zalpha's versionCode, while Jetpack only has vanilla builds and increments of 1, I just realized that we will drift eventually anyway like here 😅  Meh 🤷